src/Grid/Column/Transformer/TwigOperator.php

```
@@ -0,0 +1,105 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This source file is available under the terms of the
+ * Pimcore Open Core License (POCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (https://www.pimcore.com)
+ *  @license    Pimcore Open Core License (POCL)
+ */
+
+namespace Pimcore\Bundle\StudioBackendBundle\Grid\Column\Transformer;
+
+use Pimcore\Bundle\StudioBackendBundle\Exception\Api\TransformerException;
+use Pimcore\Bundle\StudioBackendBundle\Grid\Column\TransformerInterface;
+use Pimcore\Bundle\StudioBackendBundle\Grid\Util\AdvancedValue;
+use Pimcore\Bundle\StudioBackendBundle\Twig\TemplateGeneratorInterface;
+use function is_string;
+use function sprintf;
+
+final class TwigOperator implements TransformerInterface
+{
+    public function __construct(
+        private readonly TemplateGeneratorInterface $templateGenerator
+    ) {
+    }
+
+    public function transform(array $value, array $config): array
+    {
+        // Validate template configuration
+        if (isset($config['template']) && !is_string($config['template'])) {
+            throw new TransformerException(
+                $this->getName(),
+                sprintf(
+                    'Invalid "template" configuration (must be a string) for %s transformer. ' .
+                    'Example: "template": "{{ value|date(\'d.m.Y H:i\') }}"',
+                    $this->getKey()
+                )
+            );
+        }
+
+        $template = $config['template'] ?? '{{ value }}';
+
+        $context = [
+            'value' => $this->buildAssociativeContext($value),
+        ];
+
+        $rendered = $this->templateGenerator->generate($template, $context);
```

Catch all exception from `generate()` and  throw a TransformerException